modules/post/multi/gather/jenkins_gather.rb

```diff
@@ -0,0 +1,433 @@
+require 'nokogiri'
+require 'base64'
+require 'digest'
+require 'openssl'
+require 'sshkey'
+
+class MetasploitModule < Msf::Post
+  include Msf::Post::File
+  include Msf::Post::Linux::System
+
+  def initialize(info = {})
+    super(update_info(
+      info,
+      'Name' => 'Jenkins Credential Collector',
+      'Description' => %q(
+        This module can be used to extract saved Jenkins credentials, user
+        tokens, SSH keys, and secrets. Interesting files will be stored in
+        loot along with combined csv output.
+      ),
+      'License' => MSF_LICENSE,
+      'Author' => [ 'thesubtlety' ],
+      'Platform' => [ 'linux', 'win' ],
+      'SessionTypes' => [ %w[ shell meterpreter ] ]
+    ))
+    register_options(
+      [  OptBool.new('STORE_LOOT', [false, 'Store files in loot (will simply output file to console if set to false).', true]),
+         OptBool.new('SEARCH_JOBS', [false, 'Search through job history logs for interesting keywords. Increases runtime.', false])
+      ])
+
+    @nodes = []
+    @creds = []
+    @ssh_keys = []
+    @api_tokens = []
+  end
+
+  def report_creds(user, pass)
+    return if user.empty? || pass.empty?
+      credential_data = {
```

Comment:
I believe the indention is off, here.  The conditional statement above this line is a single line conditional statement